presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoInstaller.java

```diff
@@ -53,19 +62,20 @@ public void runCommand()
     {
         switch (packageType) {
```

Comment:
That's what I meant by "can be improved upon with Guice".

Eventually, we can have a special installation manager implementation that is configurable with the package type, and selects which of the other implementations to use.